migrations/1620954224627-add-fee-priority.js

```diff
@@ -0,0 +1,19 @@
+const { saveConfig, loadLatest } = require('../lib/new-settings-loader')
+
+exports.up = function (next) {
+  const newConfig = {
+    wallets_BTC_feeDiscount: '1'
+  }
+  return loadLatest()
+    .then(config => {
+      return saveConfig(newConfig)
+        .then(next)
+        .catch(err => {
+          return next(err)
+        })
+    })
```

Comment:
same here, each then() can return a value that will be used as the resolved value for the next then.
So instead of
```JS
return somepromise()
  .then(response => {
    return newpromise().then( innerResponse => {
      ...
    })
} )
```
we should have
```JS
return somepromise()
  .then(response => {
    return newpromise()
  })
  .then(innerResponse => {
    ...
  })
```